packages/rest/src/validation/request-body.validator.ts

```diff
@@ -190,6 +191,7 @@ function createValidator(
     {},
     {
       allErrors: true,
+      jsonPointers: true,
```

Comment:
Is it needed for ajv-errors?

Reply:
Yes, `allErrors` & `jsonPointers` both are needed by `ajv-errors` as documented in their [usage docs](https://github.com/epoberezkin/ajv-errors#usage).

When not provided, they're enabled from within `ajv-errors` at https://github.com/epoberezkin/ajv-errors/blob/master/index.js#L4-L8

Benefits of jsonPointers are listed at [ajv errors -> Templates](https://github.com/epoberezkin/ajv-errors#templates)